src/main/resources/static/data/genres.csv

```diff
@@ -1,18 +1,20 @@
 name
 Action
-Adventure
+Abenteuer
 Animation
-Comedy
-Crime and Mystery
-Documentary
+Komödie
+Krimi
+Dokumentarfilm
 Drama
+Familie
 Fantasy
-Historical
+Historie
 Horror
-Musical
-Romance
-Satire
+Musik
+Mystery
+Liebesfilm
 Science Fiction
+TV-Film
 Thriller
-War
-Western
+Kriegsfilm
```

Comment:
Sind das die Genres, die es auch in der MovieDb gibt?

Reply:
jo